parts/k8s/kubernetesinstalls.sh

```diff
@@ -220,7 +220,7 @@ installContainerd() {
     if [[ "$CURRENT_VERSION" == "${CONTAINERD_VERSION}" ]]; then
         echo "containerd is already installed, skipping install"
     else
-        CONTAINERD_TGZ_TMP=$(echo ${CONTAINERD_DOWNLOAD_URL} | cut -d "/" -f 5)
+        CONTAINERD_TGZ_TMP="cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz"
```

Comment:
This (1) removes the need for a 2nd concatenation of the `$CONTAINERD_DOWNLOAD_URL` var, and (2) is arguably more maintainable